grails-app/conf/Config.groovy

```diff
@@ -417,6 +417,11 @@ openboxes {
                     archived = true
                     order = 6
                 }
+                negativeInventory {
+                    endpoint = "/openboxes/apitablero/getProductWithNegativeInventory"
```

Comment:
Similar to the grails.serverURL, we should be using a variable for the request context path. 
```
// Default URL
grails.serverURL = "http://localhost:8080/${appName}"
```
So replace `openboxes` with `${appName}`.